packages/core/test/anyOf.test.jsx

```diff
@@ -165,6 +165,43 @@ describe('anyOf', () => {
     );
   });
 
+  it('should assign a default value and set defaults on option change for scalar types schemas', () => {
```

Comment:
Can you add a similar test for the `oneOf.test.jsx` file?

Reply:
Sure thing, PR update 👍